includes/Classifai/Providers/Watson/NLU.php

```diff
@@ -104,8 +104,16 @@ public function reset_settings() {
 	 * @return bool
 	 */
 	public function can_register() {
-		// TODO: Implement can_register() method.
-		return true;
+		$options = get_option( $this->get_option_name() );
+
+		$can_register = true;
+
+		// False where required options do not have a value saved.
+		if ( empty( $options ) || empty( $options['credentials'] ) || empty( $options['credentials']['watson_url'] ) || empty( $options['credentials']['watson_password'] ) ) {
+			$can_register = false;
```

Comment:
This prevents script loading on the Watson setting page, which is used to toggle authentication methods.

Reply:
Looks like this is resolve in: https://github.com/10up/classifai/pull/313/commits/7de00c1b492e652760bc855f83e73a2974246258

ref PR: https://github.com/10up/classifai/pull/313